contracts/oracle.sol

```diff
@@ -152,25 +103,23 @@ contract Oracle is ENSResolvable, usingProvable, Transferrable, Base64, Date, Co
         // Require that the proof is valid.
         if (valid) {
             // Parse the JSON result to get the rate in wei.
-            uint256 parsedRate = _parseIntScientificWei(_parseRate(_result));
+            uint256 parsedRate = _parseIntScientificWei(parseRate(_result));
             // Set the update time of the token rate.
             uint256 parsedLastUpdate = timestamp;
-            // Remove query from the list.
-            delete _queryToToken[_queryID];
 
-            _updateTokenRate(token, parsedRate, parsedLastUpdate);
+            _updateTokenRate(_token, parsedRate, parsedLastUpdate);
         }
     }
 
-    /// @notice Extracts JSON rate value from the response object.
+    /// @dev Extracts JSON rate value from the response object.
     /// @param _json body of the JSON response from the CryptoCompare API.
-    function _parseRate(string memory _json) internal pure returns (string memory) {
```

Comment:
I thought we name internal / private functions with an underscore, why did we change the method name ?